x-pack/plugins/apm/server/lib/metrics/by_agent/java/index.ts

```diff
@@ -17,29 +17,68 @@ import { getGcTimeChart } from './gc/get_gc_time_chart';
 
 export function getJavaMetricsCharts({
   environment,
+  kuery,
   setup,
   serviceName,
   serviceNodeName,
 }: {
   environment?: string;
+  kuery?: string;
   setup: Setup & SetupTimeRange;
   serviceName: string;
   serviceNodeName?: string;
 }) {
   return withApmSpan('get_java_system_metric_charts', async () => {
     const charts = await Promise.all([
-      getCPUChartData({ environment, setup, serviceName, serviceNodeName }),
-      getMemoryChartData({ environment, setup, serviceName, serviceNodeName }),
-      getHeapMemoryChart({ environment, setup, serviceName, serviceNodeName }),
+      getCPUChartData({
+        environment,
+        kuery,
+        setup,
+        serviceName,
+        serviceNodeName,
+      }),
```

Comment:
Wouldn't mind it if this object is defined separately, instead of inlining it in the function calls.